packages/web/pages/_document.tsx

```diff
@@ -37,12 +37,12 @@ class MyDocument extends Document<DocumentProps & { children?: ReactNode } & Cus
             rel="stylesheet"
           />
           <link
-            href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,400i,700&display=swap"
+            href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,600,700&display=swap"
```

Comment:
At some point in the near-ish future I'd like to go back through and make a few adjustments, so that we only use 300 or 400, but for now given time challenges, this seems ok to me.